.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,54 @@
+name: Feature Request
+description: Suggest an idea for this project
+labels: "enhancement"
+body: 
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to request this feature!
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe
+      description: Please provide a clear and concise description the problem this feature would solve. The more information you can provide here, the better.
+      placeholder: I'm always frustrated when...
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the proposed solution
+      description: Please provide a clear and concise description of what you would like to happen.
+      placeholder: I would like to see...
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: "Describe similar `asdf` features and why they are not sufficient"
+      description: "Describe asdf features you tried to use to accomplish what you wanted. Explain why they are insufficient for the task you were trying to accomplish."
+    validations:
+      required: true
+  - type: textarea
+    id: workarounds
+    attributes:
+      label: "Describe other workarounds you've considered"
+      description: "A clear and concise description of any alternative solutions or features you've considered."
+    validations:
+      required: true
+  - type: dropdown
+    id: importance
+    attributes:
+      label: Importance
+      description: How important is this feature to you?
+      options:
+        - nice to have
+        - would make my life easier
+        - i cannot use asdf without it
+    validations:
+      required: true
+  - type: textarea
+    id: additional-context
```

Comment:
Same thought as for the other template. All the context needed should go in the problem field.

Reply:
Not really, I was just copying over the existing templates